web/client/components/mapcontrols/annotations/FeaturesList.jsx

```diff
@@ -36,10 +37,13 @@ const FeaturesList = (props) => {
         setPopupWarning,
         geodesic,
         defaultStyles,
-        defaultPointType
+        defaultPointType,
+        onFeatureValidationCheck,
+        allGeometryIsValid
     } = props;
     const {features = []} = editing || {};
     const isValidFeature = get(props, "selected.properties.isValidFeature", true);
+    const allFeaturesAreValid = allGeometryIsValid();
```

Comment:
in general conditional flags should be named starting with a verb
or at least using correct plural form

```suggestion
    const areAllFeaturesValid = validateGeometries();
```

or 

```suggestion
    const areFeaturesValid = validateFeatures();
```
maybe we can call it validateFeatures

Reply:
Thanks, renamed into areFeaturesValid & validateFeatures